src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -57,6 +57,19 @@ public sealed class AgentSettings
 
         [DataMember(EmitDefaultValue = false)]
         public string CollectionName { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string AutoLogonUserName { get; set; }
```

Comment:
Sorry about this. I have actually defined a separate store. This is not getting used anywhere. Removed it now.

---
In reply to: [118592246](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r118592246) [](ancestors = 118592246)